### ISSUE TITLE
Incorrect WebAudio classes constructor arguments declarations in lib.es6.d.ts

### ISSUE THREAD

**Issue by cevek** [microsoft/TypeScript#14670]
```ts
declare var AudioBuffer: {
    prototype: AudioBuffer;
    new(): AudioBuffer;
}
declare var AudioBufferSourceNode: {
    prototype: AudioBufferSourceNode;
    new(): AudioBufferSourceNode;
}
// ... etc
```
## Spec
https://webaudio.github.io/web-audio-api/#idl-def-AudioBufferSourceNode
```
[Constructor(BaseAudioContext context, optional AudioBufferSourceOptions options)]
```

**TypeScript Version:**  2.3.0 / nightly

**Comment by mhegazy** [comment]
PRs welcomed. You can find more information about contributing lib.d.ts fixes at https://github.com/Microsoft/TypeScript/blob/master/CONTRIBUTING.md#contributing-libdts-fixes.

**Comment by cevek** [comment]
So, I see that link I've attached is a [W3C Editor's draft](https://webaudio.github.io/web-audio-api/) , but in [W3C working draft](https://www.w3.org/TR/webaudio/) constructors is not described.
I think after editors draft will moves to working draft, it appears on browser.webidl.xml.
But current browser implementations require these constructor arguments.

**Comment by saschanaz** [comment]
Fixed by https://github.com/Microsoft/TSJS-lib-generator/pull/471

**Comment by jakebailey** [comment]
This was fixed in #24669.
